corehq/apps/reports/templates/reports/filters/bootstrap5/multi_option.html

```diff
@@ -2,25 +2,36 @@
 {% load hq_shared_tags %}
 {% block filter_content %}
   {% if not endpoint %}
-    <select class="{{ css_class }} form-control report-filter-multi-option"
-            multiple="true"
-            data-bind="options: select_params, optionsText: 'text', optionsValue: 'val', selectedOptions: current_selection"
-            data-options="{% html_attr select.options %}"
-            data-selected="{% html_attr select.selected %}"
-            placeholder="{{ select.placeholder }}"
-            name="{{ slug }}"></select>
+    <select
+      class="{{ css_class }} form-control report-filter-multi-option"
+      name="{{ slug }}"
+      multiple="true"
+      placeholder="{{ select.placeholder }}"
+      data-bind="
+        ptions: select_params,
```

Comment:
```suggestion
        options: select_params,
```

Reply:
Assuming this blanks out the filter, I'm surprised this didn't come up in QA. What do you think?

Reply:
yeah that would be an issue! thanks

Reply:
could be the `not endpoint` condition wasn't met, which is why QA didn't see this